native/src/main/java/io/ballerina/stdlib/http/api/HttpResource.java

```diff
@@ -240,6 +287,17 @@ public static HttpResource buildHttpResource(MethodType resource, HttpService ht
         return httpResource;
     }
 
+    private void processLinkedResources(Object[] links) {
```

Comment:
Shall we rename this method to `updateLinkedResources` as per the logic ?

Reply:
Fixed in [a8c835e](https://github.com/ballerina-platform/module-ballerina-http/pull/1068/commits/a8c835e645db41d31e5f041b2fc786856153dbb3)